### PR TITLE
updated titles structure on homepage for seo

### PR DESCRIPTION
**Pull request by jchantereau**

### CODE REVIEW

**Comment by jchantereau** [comment]
@Eschults ok for u?

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -1,9 +1,10 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title><%= t(".title", default: "mon beau terroir") %></title>
+    <title><%= t(".title", default: "Mon Beau Terroir - Visitez et dégustez chez les producteurs du terroir") %></title>
     <%= csrf_meta_tags %>
-    <meta name="description" content="DEGUSTEZ ET PARTAGEZ CHEZ LES PRODUCTEURS. Réservez vos visites chez nos producteurs sélectionnés et passionnés">
+    <meta name="description" content="VISITEZ, DEGUSTEZ ET PARTAGEZ CHEZ LES PRODUCTEURS. Avec Mon Beau Terroir, réservez vos visites chez nos producteurs sélectionnés et passionnés">
```
Comment (Eschults) [inline app/views/layouts/application.html.erb]: `content="<%= t(".description", default: "...") %>"` would be better for a future internationalization

Comment (jchantereau) [inline app/views/layouts/application.html.erb]: true, done

**Comment by Eschults** [comment]
good, maybe check there's not already a `description` key in yml locales
the best process for handling metas (especially when you work with multiple locales) is described here: https://www.lewagon.com/fr/blog/tuto-setup-metatags-rails
just check there's no `description` yet and I'll merge

**Comment by jchantereau** [comment]
no description key in yml, I see the method, I think it will be consistent if we need to add more metas but for now I think it's just adding troubles for a simple meta desc.

Thx for the merge

**Pull request merged**